docs/src/main/paradox/jdbc.md

```diff
@@ -1,8 +1,246 @@
-# Offset in relational DB with JDBC
+# Offset in a relational DB with JDBC
 
-Not implemented yet, see [issue #23](https://github.com/akka/akka-projection/issues/23).
+The @apidoc[JdbcProjection] has support for storing the offset in a relational database using JDBC.
+
+The source of the envelopes can be @ref:[events from Akka Persistence](eventsourced.md) or any other `SourceProvider`
+with supported @ref:[offset types](#offset-types).
+
+A @apidoc[JdbcHandler] receives a @apidoc[JdbcSession] instance and an envelope. The `JdbcSession` provides the means to access an open JDBC connection that can be used to process the envelope. The target database operations can be run in the same transaction as the storage of the offset, which means that @ref:[exactly-once](#exactly-once)
+processing semantics is supported. It also offers @ref:[at-least-once](#at-least-once) semantics.
+
+## Dependencies
+
+To use the JDBC module of Akka Projections add the following dependency in your project:
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.lightbend.akka
+  artifact=akka-projection-jdbc_$scala.binary.version$
+  version=$project.version$
+}
+
+Akka Projections require Akka $akka.version$ or later, see @ref:[Akka version](overview.md#akka-version).
+
+@@project-info{ projectId="akka-projection-jdbc" }
+
+
+### Transitive dependencies
+
+The table below shows `akka-projection-jdbc`'s direct dependencies, and the second tab shows all libraries it depends on transitively.
+
+@@dependencies{ projectId="akka-projection-jdbc" }
+
+## Defining a JdbcSession
+
+Before using Akka Projection JDBC you must implement a `JdbcSession` @scala[trait]@java[interface]. `JdbcSession` is used to open a connection and start a transaction. A new `JdbcSession` will be created for each call to the handler. At the end of the processing, the transaction will be committed (or rolled back). 
+
+When using `JdbcProjection.exactlyOnce`, the `JdbcSession` that is passed to the handler will be used to save the offset behind the scenes. Therefore, it's extremely important to disable auto-commit (eg: `setAutoCommit(false)`), otherwise the two operations won't participate on the same transaction.  
+
+Scala
+:  @@snip [JdbcProjectionDocExample.scala](/examples/src/test/scala/docs/jdbc/JdbcProjectionDocExample.scala) { #jdbc-session-imports #jdbc-session }
+
+Java
+:  @@snip [JdbcProjectionDocExample.java](/examples/src/test/java/jdocs/jdbc/JdbcProjectionDocExample.java) { #jdbc-session-imports #jdbc-session }
+
+
+@@@ div { .group-java }
```

Comment:
Maybe we should show this section also when Scala is selected? It highlights that it can be integrated with other db libraries. We can still only have the snip in Java tab.